src/components/TodoApp/index.js

```diff
@@ -0,0 +1 @@
+export * from './TodoApp';
```

Comment:
why is it js file? should be .ts)

Reply:
can you please explain me why it should be ts but not js?